src/main/java/seedu/address/ui/EventListPanel.java

```diff
@@ -0,0 +1,49 @@
+package seedu.address.ui;
+
+import java.util.logging.Logger;
+
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.layout.Region;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.event.Event;
+
+/**
+ * Panel containing the list of persons.
+ */
+public class EventListPanel extends UiPart<Region> {
+    private static final String FXML = "EventListPanel.fxml";
+    private final Logger logger = LogsCenter.getLogger(EventListPanel.class);
+
+    @FXML
+    private ListView<Event> eventListView;
+
+    /**
+     * Creates a {@code PersonListPanel} with the given {@code ObservableList}.
+     */
+    public EventListPanel(ObservableList<Event> eventList) {
+        super(FXML);
+        eventListView.setItems(eventList);
+        eventListView.setCellFactory(listView -> new EventListViewCell());
+    }
+
+    /**
+     * Custom {@code ListCell} that displays the graphics of a {@code Person} using a {@code PersonCard}.
+     */
+    class EventListViewCell extends ListCell<Event> {
+        @Override
+        protected void updateItem(Event event, boolean empty) {
```

Comment:
Not sure I understand the logic here or how it's used - can explain?

Reply:
It is used to set the graphic in the ListCell<Event> to be that of an EventCard.